### DISCUSSION TITLE
Running Dokploy behind revers proxy!

### DISCUSSION THREAD

**Discussion by channchetra** [Dokploy/dokploy#194] [Q&A] [answered]
I'm trying Dokploy on a private VPS behind a Reverse Proxy server. Since Dokploy is on the private VPS, it doesn't have a public IP address, but everything is working fine.

I pointed my domain to the Reverse Proxy, and the proxy forwards traffic to the_privateIP:3000 which allows me to access Dokploy using my domain. I do the same with all my applications, by reversing traffic to their specific published ports.

However, I'm having trouble Enabling Self Hosted Registry. I've pointed registry.mydomain.com to the Proxy Server, but I'm unsure which specific ports for the registry I need to add in the proxy configuration.

Thank you for your help.

**Comment by Siumauricio** [comment, marked as answer]
Hey, the ports of registry by default are 5000 you can check here

https://github.com/Dokploy/dokploy/blob/canary/server/setup/registry-setup.ts#L55C18-L55C22

**Reply by channchetra** [reply]
> Hey, the ports of registry by default are 5000 you can check here
> 
> https://github.com/Dokploy/dokploy/blob/canary/server/setup/registry-setup.ts#L55C18-L55C22

Thank you it's working fine now 🙏
